src/satellite_consumer/process.py

```diff
@@ -87,102 +84,130 @@ def _map_scene_to_dataarray(
         the scan finished, not the time it started. I don't dare change
         it in order to stay consistent with all the historical data,
         but it is important to be aware of it.
-
-    Args:
-        scene: The satpy.Scene containing the satellite data.
-        crop_region_geos: Optional bounds to crop the data to, in the format
     """
+    # Convert the Scene to a DataArray, filtering to the desired data variables
     for channel in scene.wishlist:
-        # Drop unwanted variables
         scene[channel] = scene[channel].drop_vars("acq_time", errors="ignore")
 
-    # Convert the Scene to a DataArray
-    da: xr.DataArray = scene.to_xarray_dataset().to_array(name="data")
+    with warnings.catch_warnings(action="ignore"):
+        ds: xr.Dataset = (
+            scene.to_xarray_dataset() # type: ignore
+            .drop_vars("acq_time", errors="ignore")[[c.name for c in channels]]
```

Comment:
```suggestion
            .drop_vars("acq_time", errors="ignore")
```

I think we don't need the `[[c.name for c in channels]]` line. IIRFC we had hoped that doing that would be an alternative to using ` .drop_vars("acq_time", errors="ignore")`. I think using both is redundant